<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="LoginPage">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>250</y>
        <width>481</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Login</string>
      </property>
      <widget class="QLineEdit" name="usernameLine">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordLine">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>120</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLabel" name="username">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Login Username:</string>
       </property>
      </widget>
      <widget class="QLabel" name="password">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Login Password:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clearButton">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>160</y>
         <width>141</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QPushButton" name="LoginButton">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>160</y>
         <width>141</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QPushButton" name="guestButton">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>200</y>
         <width>341</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Continue as Guest</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label5">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>341</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="AdminLandingPage">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;THIS IS THE ADMIN LANDING PAGE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="adminReturn">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>191</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Return to Main Menu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="GuestLandingPage">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;THIS IS THE GUEST LANDING PAGE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="guestReturn">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>140</y>
        <width>191</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Return to Main Menu</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
